src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -110,6 +110,7 @@
                 Command="{Binding OpenPROnGitHub, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:PullRequestListView}}}"
                 CommandParameter="{Binding Number}"
                 Content="{Binding Number}"
+                ToolTip="View Pull Request on GitHub"
```

Comment:
We already do the distinction between "GitHub" and "explicit server name" in the TE home section title and the Connect section title, and there's already code readily available to do this, if that's what's worrying you 😄